alibuild_helpers/build.py

```diff
@@ -1112,6 +1114,30 @@ def doBuild(args, parser):
       """)
       buildErrMsg += "".join("\n  ( cd %s && git pull --rebase )" % dp for dp in updatablePkgs)
 
+    # Gather build info for the error message
+    try:
+      safe_args = {
+        "pkgname", "defaults", "architecture", "forceUnknownArch",
+        "develPrefix", "jobs", "noSystem", "noDevel", "forceTracked", "plugin",
+        "disable", "annotate", "onlyDeps", "docker", "docker_extra_args",
```

Comment:
don't we potentially have password based authentication possible via docker_extra_args, remote store?